src/docs/release/breaking-changes/test-text-input.md

```diff
@@ -3,67 +3,73 @@ title: TestTextInput state reset
 description: TestTextInput will have its state reset between tests.
 ---
 
-# TestTextInput state will be reset between tests
+## Summary
+
+The state of a `TestTextInput` instance,
+a stub for the system's onscreen keyboard,
+is now reset between tests.
 
 ## Context
 
 The Flutter test framework uses a class called `TestTextInput` to track and
 manipulate editing state in a widgets test. Individual tests can make calls
 that modify the internal state of this object, sometimes indirectly (such as
-by setting their own handlers on `SystemChannels.textInput`). Subsequent tests
-may then check the state of `WidgetTester.testTextInput` and get unexpected
-values.
+by setting their own handlers on `SystemChannels.textInput`).
+Subsequent tests might then check the state of `WidgetTester.testTextInput`
+and get unexpected values.
 
 ## Description of change
 
-Reset the state of `WidgetTester.testTextInput` before running a `testWidgets`
-test.
+The state of the `WidgetTester.testTextInput` 
+is now reset before running a `testWidgets` test.
 
 ## Migration guide
 
-Tests that relied on dirty state from a previously run testwill have to be
-updated. For example, this test:
-
-```dart
-    expect(tester.testTextInput.editingState['text'], isEmpty);
-```
+Tests that relied on dirty state from a previously run test must be
+updated. For example, the following test, 
+from `packages/flutter/test/material/text_field_test.dart` in the
+`'Controller can update server'` test, previously passed because
+of a combination of dirty state from previous tests and a
+failure to actually set state in cases where it should have been set.
 
-from `packages/flutter/test/material/text_field_test.dart` in the `'Controller
-can update server'` test previously passed because of a combination of dirty
-state from previous tests and a failure to actually set state in cases where
-we should have been.
-
-### Before
+Code before migration:
 
 In a `widgetsTest`, before actually changing text on a text editing widget,
 this call might have succeeded:
 
+<!-- skip -->
 ```dart
     expect(tester.testTextInput.editingState['text'], isEmpty);
 ```
 
-### After
+Code after migration:
 
-Either remove the call entirely, or consider:
+Either remove the call entirely, or consider using the
+following to assert that the state hasn't been modified yet:
 
+<!-- skip -->
 ```dart
     expect(tester.testTextInput.editingState, isNull);
 ```
 
-to assert that the state hasn't been modified yet.
-
 ## Timeline
 
-This change is being proposed for December of 2019.
+TBA: This change is being proposed for December of 2019.
```

Comment:
This change landed already (in December)

Reply:
Otherwise this file LGTM

Reply:
This should be `v1.13.5` as the first version where this showed up.

Reply:
(unless we truly need the `pre` version that it showed up in before that, which would be really hard for anyone to sync to anyway)